mkdocs.yml

```diff
@@ -68,14 +68,14 @@ nav:
         - User Guide: "user-guide/patch-maker/user-guide.md"
         - Troubleshooting: "user-guide/patch-maker/troubleshooting.md"
   - Contributor Guide:
-      - Getting Started: "contributor-guide/getting-started.md"
+      - "Getting Started & Python Conventions": "contributor-guide/python.md"
```

Comment:
Maybe just call this "Getting Started" and make the file "contributor-guild/getting-started.md"?